### ISSUE TITLE
Which USB cable was it tested with?

### ISSUE THREAD

**Issue by andershedberg** [eggsampler/ESP32-S2-Breakout#5]
Hi, having a bit of a problem with my version I wonder what kind of USB connection this was tested with? 

If I use a USB A to USB C cable I get power on the board. ( but no connection )
If I use a USB C to USB C cable I do not get power to the board. ( and no connection of course )

It seems I got the wrong USB C connector on my board, they look all the same, but there are minor differences, so I will look into that to see if it really has a connection even though it looks like that.

**Comment by eggsampler** [comment]
This is the connector I used: https://lcsc.com/product-detail/USB-Connectors_Korean-Hroparts-Elec-TYPE-C-31-M-12_C165948.html

Which one did you use?

**Comment by andershedberg** [comment]
This one, as I thought it was faster to ship from the US/Mouser ( Don't, the ESPRESSIF Modules are for US only and can't be exported to Europe for some paperwork... so I got those from a friend. ). USB - Unfortunately the front metal pins are wider than the original and there is no drawing in the datasheet to tell. https://www.jae.com/direct/topics/topics_file_download/?topics_id=91781&ext_no=04&_lang=en&v=2020041313411436908609

**Comment by andershedberg** [comment]
Sorry, back to the question - as I experience problems with a pure USB C to USB C cable. Did you try it with that and the board got power? That only happens with my USB C to USB A cable.

**Comment by eggsampler** [comment]
I've just gone through and tested with a USB-C to USB-C macbook pro cable and it powered on and worked from both the charger, and uploaded code fine from the macbook. Did you end up having any luck?

**Comment by andershedberg** [comment]
Great news! Even though I could not get it to work :-/ 

I cut a USB A mouse-cable and soldered it directly to the ESP32 side of things and then it worked and could be flashed etc. I could not get the USB C to even deliver power to it so something was not right from my HP and Acer laptops. I did not spend more time with it, as it might have been my soldering, my a bit modified board layout, wrong component rotation or the USB C board connector which I thought were standardized but I ended up with the wrong one ( a bit too long metal case legs, and there are no measurments for that in the datasheets, really strange ). I ended up doing your older micro-USB design and will mount it tomorrow. If I get more time I might try to mount an extra USB-C board as I have now got some better soldering equipment as well and know that it should work :-)
